tests/components/fritzbox/test_light.py

```diff
@@ -95,7 +96,7 @@ async def test_turn_on(hass: HomeAssistant, fritz: Mock):
     assert await hass.services.async_call(
         DOMAIN,
         SERVICE_TURN_ON,
-        {ATTR_ENTITY_ID: ENTITY_ID, ATTR_BRIGHTNESS: 100, ATTR_COLOR_TEMP: 300},
+        {ATTR_ENTITY_ID: ENTITY_ID, ATTR_BRIGHTNESS: 100, ATTR_COLOR_TEMP_KELVIN: 300},
```

Comment:
300 kelvin, even 2700 is minimum? 🤔

Reply:
Hm, I guess it is a typo and should be 3000. Makes no difference in this test, but you are right, the test should better use a valid value.

Reply:
maybe we should also check the result state after the service call 🤔

Reply:
I added asserts to check the arguments in the device call.